imdbapi.class.php

```diff
@@ -201,6 +201,16 @@ private function removeAccents($string) {
         return $string;
     }
 
+    private function validateGetData($returnedData) {
+        if (array_key_exists('error', $returnedData)) {
```

Comment:
I think it will be better to throw exceptions rather than HTML. Reason being this tool could be using in CLI and in webserver ways rather than just webserver (html rendering)

Its a library and we would want to leave the user how they want to handle/display/ignore errors.

Throwing an exception here would make more sense 👍